include_core/omrport.h

```diff
@@ -1766,6 +1766,42 @@ typedef struct OMRProcessorDesc {
 #define OMR_FEATURE_X86_AVX512BW            96 + 30 /* AVX512 Byte and Word */
 #define OMR_FEATURE_X86_AVX512VL            96 + 31 /* AVX512 Vector Length */
 
+/*
+ * Vol. 2 3-197 Table 3-8. Structured Feature Information Returned in the ECX Register by CPUID instruction
```

Comment:
"Vol 2" is fine, but the page numbers change very regularly between revisions (and the table numbering might as well) that it doesn't make sense to include them in a comment.